<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpKeyboardWidget</class>
 <widget class="QWidget" name="SpKeyboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Screen-keyboard</string>
  </property>
  <widget class="Line" name="line1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="bdel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;----</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="benter">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>180</y>
     <width>70</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bh">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bb">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ba">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b0">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bs">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bu">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ben">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ã‘</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br3">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="by">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bprod">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bdiv">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brmin">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br8">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bbloq">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Bloq
Num</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bdot">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f12">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bp">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="esc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bcut">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bz">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bf">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bmsg">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blalt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brctrl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ctrl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bx">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bm">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brplus">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>180</y>
     <width>51</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br5">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bk">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bq">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bspace">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>390</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="blsh">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br0">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>300</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br7">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bc">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bg">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bv">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brdot">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bi">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brsh">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>40</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bo">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Âº
Âª</string>
   </property>
  </widget>
  <widget class="QPushButton" name="be">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f11">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bw">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blmi">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br6">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bralt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AltGr</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bj">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f10">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="blctrl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ctrl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f9">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>F4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br9">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brenter">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>260</y>
     <width>51</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>Intro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>220</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="br1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>260</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_line">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>10</y>
     <width>950</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>230</y>
     <width>70</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>ocultar teclado y salir guardando</string>
   </property>
   <property name="whatsThis">
    <string>ocultar teclado y salir</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>120</y>
     <width>70</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Borrar la linea escrita</string>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bmay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Caps Lock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bint">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
